tests/test_consumer.py

```diff
@@ -346,35 +345,28 @@ def test_dead_letter_step() -> None:
 
 def test_metrics_writing_e2e() -> None:
     from snuba.datasets.storages.metrics import (
-        counters_storage,
```

Comment:
This and sets_storage are no longer in use, and we're never going to set `settings.WRITE_METRICS_AGG_DIRECTLY = True`